app-backend/batch/src/main/resources/application.conf

```diff
@@ -119,4 +119,7 @@ dropbox {
 
 auth0 {
   systemUser = ${?AUTH0_SYSTEM_USER}
+  clientId = ${?AUTH0_MANAGEMENT_CLIENT_ID}
```

Comment:
Looks like these are going to need to be added in the deployment repository to environment variables in the batch jobs for these.